### PR TITLE
alert the user when bootstrap failed

### PR DESCRIPTION
**Pull request by MaximilianMeister**
otherwise the user is left without any clue what to do

Signed-off-by: Maximilian Meister <mmeister@suse.de>

@vitoravelino could you please have a look if this makes sense, or if it can be done better?

@kiall @ereslibre would it be easy to get the error(s) of the orchestration to be more specific in the flash message what happened? i believe it might be not that trivial to filter the responsible error and then condense it in a message that fits into a flash message...

### CODE REVIEW

File: app/assets/javascripts/dashboard.js [hunk 1]
```diff
@@ -244,6 +244,13 @@ MinionPoller = {
     $("#download-kubeconfig").attr("disabled", !enabled);
   },
 
+  alertFailedBootstrap: function(message) {
+    if ($('.alert-message span').is(':empty'))
```
Comment (ereslibre) [inline app/assets/javascripts/dashboard.js]: Is missing braces the recommended style?

**Comment by ereslibre** [comment]
> @kiall @ereslibre would it be easy to get the error(s) of the orchestration to be more specific in the flash message what happened? i believe it might be not that trivial to filter the responsible error and then condense it in a message that fits into a flash message...

This is really tricky at the moment (to fetch the relevant information). We can do this when we have https://github.com/kubic-project/velum/pull/315, https://github.com/kubic-project/velum/pull/299 and https://github.com/kubic-project/velum/pull/295.

IMHO with all those tools we will be in a much better position to notify the user *what* went wrong in a very clear way.

File: app/assets/javascripts/dashboard.js (hunk 1 above)
Comment (vitoravelino) [reply app/assets/javascripts/dashboard.js]: Use curly braces always.

Comment (MaximilianMeister) [reply app/assets/javascripts/dashboard.js]: fixed

**Comment by vitoravelino** [comment]
I would probably use a red alert if all the nodes or at least 50% + 1 did not bootstrap. Not sure if it's necessary. Just an idea that came up. 😄

**Comment by MaximilianMeister** [comment]
> I would probably use a red alert if all the nodes or at least 50% + 1 did not bootstrap. Not sure if it's necessary. Just an idea that came up.

good point, it depends if the cluster use usable or not i would say. but i think a generic warning that something needs to be investigated is already fine

**Comment by vitoravelino** [comment]
> good point, it depends if the cluster use usable or not i would say. but i think a generic warning that something needs to be investigated is already fine

Makes sense. We can improve that after we get more info about the cluster as mentioned by @ereslibre.

LGTM 😃

**Comment by flavio** [comment]
I feel the "look at at logs to see what went wrong" is a bit of a joke right now. It's really hard to get down them until all the PR mentioned by @ereslibre are merged (and some additional work is done).

However I don't see any harm in having this code merged.

**Comment by flavio** [comment]
@eotchi what do you think about that?

**Comment by MaximilianMeister** [comment]
can we improve the message a bit?

**Comment by eotchi** [comment]
Just an idea, feel free to ignore. What if instead of saying to look at the logs we display instructions on how to run supportconfig and where to find the generated archives until we have something better?

**Pull request merged**